charts/ds389/templates/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
This file is not needed and could create problem (Helm can create the namespace if needed)